client/my-sites/plans/current-plan/current-plan-thank-you-card/paid-plan-thank-you-card.js

```diff
@@ -1,97 +1,206 @@
 /**
  * External dependencies
  */
-import React, { Component, Fragment } from 'react';
 import { connect } from 'react-redux';
+import { get } from 'lodash';
 import { localize } from 'i18n-calypso';
+import { parse as parseUrl } from 'url';
+import Gridicon from 'gridicons';
+import React, { Component, Fragment } from 'react';
 
 /**
  * Internal dependencies
  */
+import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteFileModDisableReason } from 'lib/site/utils';
+import { isJetpackSiteMainNetworkSite, isJetpackSiteMultiSite } from 'state/sites/selectors';
+import { preventWidows } from 'lib/formatting';
+import { SETTING_UP_PREMIUM_SERVICES } from 'lib/url/support';
+import { untrailingslashit } from 'lib/route';
 import Button from 'components/button';
-import Card from 'components/card';
 import getJetpackProductInstallProgress from 'state/selectors/get-jetpack-product-install-progress';
 import JetpackProductInstall from 'my-sites/plans/current-plan/jetpack-product-install';
 import ProgressBar from 'components/progress-bar';
-import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import ThankYouCard from './thank-you-card';
+
+const INSTALL_STATE_COMPLETE = 1;
+const INSTALL_STATE_UNCOMPLETE = 2;
 
 export class PaidPlanThankYouCard extends Component {
-	getMyPlanRoute() {
-		const { siteSlug } = this.props;
+	render() {
+		const {
+			fileModDisableReason,
+			hasMinimumJetpackVersion,
+			installProgress,
+			installState,
+			isSiteMainNetworkSite,
+			isSiteMultiSite,
+			site,
+			translate,
+		} = this.props;
 
-		return `/plans/my-plan/${ siteSlug }`;
-	}
+		const securityIllustration = '/calypso/images/illustrations/security.svg';
+		const fireworksIllustration = '/calypso/images/illustrations/fireworks.svg';
 
-	render() {
-		const { progressComplete, translate } = this.props;
+		// Jetpack is too old
+		if ( ! hasMinimumJetpackVersion ) {
+			// Link to "Plugins" page in wp-admin
+			let wpAdminPluginsUrl = get( site, 'options.admin_url' );
+			wpAdminPluginsUrl = wpAdminPluginsUrl
+				? untrailingslashit( parseUrl( wpAdminPluginsUrl ).pathname ) + '/plugins.php'
+				: undefined;
+
+			return (
+				<ThankYouCard
+					illustration={ fireworksIllustration }
+					showContinueButton={ ! wpAdminPluginsUrl }
+					showHideMessage={ wpAdminPluginsUrl }
+					title={ translate( 'Thank you for your purchase!' ) }
+				>
+					<p>
+						{ preventWidows(
+							translate(
+								'Unfortunately, we are unable to set up your plan because your site has an older version of Jetpack. Please upgrade Jetpack.'
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'We are unable to set up your plan because your site has an older version of Jetpack. Please upgrade Jetpack.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
If you'd like to keep the "Unfortuantely", maybe you can also update (at the same time) the version without that word so we don't end up with 2 different strings? The other (already translated version) is located here:
https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx#L297

Reply:
Thanks, good idea. We'll redirect people away from that flow today and prolly remove the code soon but happy to update there now, too.

Reply:
Great, thanks. Once this lands both strings will be updated and the translations should be preserved.